### ISSUE TITLE
Re #76 - no location name on record.

### ISSUE THREAD

**Issue by MNSmith** [NERC-CEH/irecord-app#77]
My earlier issue #76 was closed, but I don't think the answers addressed the particular issue about not allowing a record to be uploaded to iRecord without a location name.

Having looked at #45 and the other comments, neither addresses the point of my original query.  Having the ability to lock a location name is useful, but of no use in this instance if no location name is given.

While I agree that a location name is a useful aid to verification, I am still looking for this app to be essentially a quick way of recording in the field.  It has already been discussed that a) entering location names on a small phone screen can be tricky and time consuming and b) recorders may not know where the nearest relevant place name is to the record, particularly if they are in unfamiliar or sparsely populated areas.

I would really like to see this app be just a "point and click" recording method in the field - here I am, I see species X - I record it and the GPS provides the location.  If I have time or the knowledge, I can, if I want, add the location name at the point of record.  However, what I had hoped is that records without the location name could be uploaded to iRecord and any location name details could be added later, when perhaps it is more easy to look at the map on screen and give a location name.

If iRecord is going to require a location name (and some of the names I have seen during verification are essentially useless), then perhaps these uploaded records could be "held" at the iRecord end with a flag on the record visible to the recorder saying "Needs a location name in order to be verified - please edit the record" or similar.  This gives the recorder the chance to sort things out and add info which he may not know in the field.   At present I have a record without a location name "stuck" on the app as I cannot upload it unless I start fiddling with mapping on a small phone screen.

**Comment by kazlauskis** [comment]
The default iRecord website form does not allow records without a location name and it should be the same with the app. Some schemes might want it differently which should be addressed with #14, but for the time being I feel the location name should be a requirement. 

Flagging a record as 'Needs location name' would force the users to use both the App and the website to make a valid record, which would put many people off.

I will leave this open for a discussion from other users.

**Comment by BlueCarbon7** [comment]
I agree with Matt that there should be an option to use the app as a 'notebook' with addition of location name via the website (e.g. where details can be added on a 'real' keyboard), as this permits faster field recording, and faster data entry later on.

Maybe a 'settings' option is needed to permit 'draft' upload of records that are missing mandatory iRecord' data fields, so they can be edited in iRecord before being submitted. I realise this might also require changes in iRecord to create a 'draft' record, but it would be very useful. This follows the 'cloud' concept Karolis has mentioned before, where data is synched across different platforms.

Typing location names accurately on a phone screen can be time consuming and frustrating (especially when auto-correct is on!).

**Comment by johnvanbreda** [comment]
For information, Indicia supports a record status flag of I (in progress - i.e, still in draft form). So we could make use of that. There would still need to be some changes on iRecord to be able to easily list your records that still need further work.
